docs/design/StratisSoftwareDesign.lyx

```diff
@@ -1001,6 +1001,21 @@ Most stratisd D-Bus methods require root permissions.
 Prevent Spoofing
 \end_layout
 
+\begin_layout Subsection
+Querying stratisd state via D-Bus
+\end_layout
+
+\begin_layout Standard
+Instead of providing overly verbose stratisd internal state logging, stratisd
+ exposes a reporting interface.
+ This allows users to query the state of stratisd's internal data structures
+ for debugging and error reporting purposes.
```

Comment:
You can drop the word "purposes" and then "debugging" and "error reporting" can be nouns instead of adjectives. Also, see above.